ingest/scripts/extract-genotype

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python3
+"""
+Parse ENTREZ records in NDJSON format and extracts the genotype
+"""
+
+from augur.io.json import load_ndjson
+import argparse
+from collections.abc import Generator
+from collections import defaultdict
+import re
+from pprint import pp
+from sys import stderr
+from csv import DictWriter
+
+def clean_up_genotype(genotype: str) -> str|None:
+    if genotype.startswith("RV"):
+        genotype = genotype[2:]
+    if genotype.endswith(';'):
+        genotype = genotype[:-1]
+
+    # coerce to uppercase -- NOTE: John A mentioned there is semantic meaning
+    # in case here. TODO XXX
+    genotype = genotype.upper()
+
+    # Some strain names encode "CRS" in what we interpret to be the genotype...
```

Comment:
This is a designation that the sample came from a child with [congenital rubella syndrome](https://en.wikipedia.org/wiki/Congenital_rubella_syndrome).